joplin/api/schema.py

```diff
@@ -193,6 +194,117 @@ class Meta:
         model = LocationPageRelatedServices
         interfaces = [graphene.Node]
 
+class EventPageRemoteLocation(graphene.ObjectType):
```

Comment:
I remember we spoke about this verbally, but I think a docstring or extended comment here would be helpful, to explain why we aren't using the DjangoObjectType but wrote a 'custom schema'.

Also see my note below about the name of the data model

Reply:
addressed in new comments